squad_client/commands/submit.py

```diff
@@ -174,4 +174,6 @@ def run(self, args):
             attachments=args.attachments,
         )
 
+        logger.info(f"TESTRUN_ID {testrun_id}")
```

Comment:
Nice! 

@roxell I had a chat with Katie earlier and we agreed that when you call `squad-client submit` to upload your mm_tests attachments to SQUAD, squad-client would spit the testrun id on the terminal like this. Then you would use that id later in the pipeline to retrieve the same files back from SQUAD. Just wanted to check with you to make sure we're all on the same page